rules.ini

```
@@ -205,3 +205,4 @@ GameCenter = (?:^|/)GameCenterDistribs\.xml$
 RGL = (?:^|/)Rockstar-Games-Launcher\.exe$
 Ubisoft_Connect[] = (?:^|/)UbisoftConnectInstaller\.exe$
 Ubisoft_Connect[] = (?:^|/)UplayInstaller\.exe$
+WGC = (?:^|/)wgcs_api\.exe$
```

```suggestion
Wargaming_GameCenter = (?:^|/)wgcs_api\.exe$
```

(should probably change RGL too).